src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const ACTIVE = 'ACTIVE';
+const NOT_ACTIVE = 'NOT_ACTIVE';
+
+const actionForCategory = () => ({ type: NOT_ACTIVE });
+
```

Comment:
- [ ] Kindly ensure that this function exports the correct action type defined above.

- [ ] **[OPTIONAL]** I would recommend you rename this function to something more meaningful like `checkStatus`, for example.